PostHog/PostHogSDK.swift

```diff
@@ -213,7 +226,7 @@ let maxRetryDelay = 30.0
     }
 
     private func getRegisteredProperties() -> [String: Any] {
-        guard let props = storage?.getDictionary(forKey: .registerProperties) as? [String: String] else {
+        guard let props = storage?.getDictionary(forKey: .registerProperties) as? [String: Any] else {
```

Comment:
I am unsure if `$groups` is always `String: String` or `String: Any`, apparently its the former but JS does `String: Int` as well for some reason.

Reply:
I guess we don't stop people from passing a number like `{ project: 123 }` but I'm not sure what actual effect that has downstream...

Reply:
Will keep `String: Any` for safety and let ingestion decide if its ok or not, the SDK only sets `String: String` anyway, this would happen if somebody passes `$groups` manually.